### PR TITLE
New Commcare Adaptor Doc

### PR DESCRIPTION
**Pull request by christad92**
New adaptor overview documentation

### CODE REVIEW

File: adaptors/commcare.md [hunk 1]
```diff
@@ -1,219 +1,204 @@
...
-6. Choose "JSON" as Payload Format
-7. Select which case type(s) you want to forward, for example "patient"
+5. Select `POST` HTTP Request Method
+6. Choose `JSON` as Payload Format
+7. Select which case type(s) you want to forward, for example `patient`
 8. Exclude any (eg. test) users
 
 ![Cases](/img/commcare_forward_cases.png)
 
-## Data Forwarding and OpenFn Workflow Design
```
Comment (mtuchi) [inline adaptors/commcare.md:104]: @christad92 i think some of these information might be useful to someone. I will suggest we turn them into a community post and it's okay if we can have a banner saying this is outdated information.

**Review by mtuchi — COMMENTED** [review]
@christad92 looks good to me, I left one suggestion if we could create a community post using the old content in `Integration tips & Unique Identifiers`

**Review by ritazagoni — APPROVED** [review]
Looking great. Made a few small edits.

File: adaptors/commcare.md [hunk 2]
```diff
@@ -1,219 +1,204 @@
...
+synchronization and integration scenarios.
 
-## APIs & Integration Options
+#### Some use cases include:
 
-CommCare offers a number of integration options for extracting and/or loading
-data to and from CommCare HQ.
+- Syncing data from CommCare to DHIS2: This integration allows for seamless
+  transfer of health data collected in CommCare to DHIS2 for comprehensive
+  analysis, visualization and reporting.
```
Comment (AishaHassen) [inline adaptors/commcare.md:45]: Suggestions to add:
- Syncing data from CommCare to a Shared Health Repository: Health facilities that use CommCare as an EMR can integrate with national shared health repositories to send individual patient data.

**Review by AishaHassen — APPROVED** [review]

**Comment by christad92** [comment]
> @christad92 looks good to me, I left one suggestion if we could create a community post using the old content in `Integration tips & Unique Identifiers`

Thanks! We should build ID management in CommCare into a different tutorial/community post.  Please can you approve the PR if everything is good for you?

**Review by mtuchi — APPROVED** [review]

**Comment by aleksa-krolls** [comment]
hey @christad92 please do _not_ delete any content from old CommCare docs (e.g., info on unique identifiers) unless you immediately move it somewhere else, otherwise we risk forgetting about it. I've added back this content for now, because I don't want to lose valuable content and learnings. Let's please discuss the question on "where to put other helpful integration tips" in our next adaptor sync.

**Review by aleksa-krolls — APPROVED** [review]

File: adaptors/commcare.md (hunk 1 above)
Comment (aleksa-krolls) [reply adaptors/commcare.md:104]: Added back for now until we figure out where to put this info

**Pull request merged**